zcash_client_sqlite/src/wallet/init.rs

```diff
@@ -365,16 +367,15 @@ mod tests {
         }
 
         let expected_views = vec![
-            // v_sapling_shard_unscanned_ranges
+            // v_sapling_shard_scan_ranges
             format!(
-                "CREATE VIEW v_sapling_shard_unscanned_ranges AS
-                WITH wallet_birthday AS (SELECT MIN(birthday_height) AS height FROM accounts)
+                "CREATE VIEW v_sapling_shard_scan_ranges AS
```

Comment:
What happens to the old view? Or did we not release any version with the old view?